api/MediaRecorderErrorEvent.json

```diff
@@ -0,0 +1,101 @@
+{
+  "api": {
+    "MediaRecorderErrorEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaRecorderErrorEvent",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": false
```

Comment:
Interesting -- I looked in the Chromium source and the only references to this interface name were in an HTML file. I also apparently incorrectly thought false meant "unknown".

Reply:
So, looking at the bug for implementing MediaRecorder, they initially implemented MediaRecorderErrorEvent, then removed that and used just ErrorEvent instead. The spec went through a fluctuation where it vacillated back and forth between the two, IIRC:

https://bugs.chromium.org/p/chromium/issues/detail?id=262211#c82

The discussion on the spec itself resolved with the decision to have the MediaRecorderErrorEvent interface with the required error property, as I documented, but Chrome does not yet appear to implement this that I can see; they also do not have a MediaRecorderErrorEvent IDL file in their source tree, and their code is generating a generic error event rather than the specific type.

That said, they do throw an event that includes the error property. I'll change these to version_added false and a note.